docs/changelog.asciidoc

```diff
@@ -361,6 +361,9 @@ The client API leaks HTTP-related notions in many places, and removing them woul
 
 This could be a rather big breaking change, so a double solution could be used during the 8.x lifecycle. (accepting body keys without them being wrapped in the body as well as the current solution).
 
+To convert code from 7.x, you just need to remove the `body` parameter in all the endpoints request.
```

Comment:
Can you please eliminate the word just? https://developers.google.com/style/word-list#just